packages/thirdweb/src/react/native/ui/components/Address.tsx

```diff
@@ -0,0 +1,55 @@
+import Clipboard from "@react-native-clipboard/clipboard";
+import { useEffect, useState } from "react";
+import { StyleSheet, TouchableOpacity } from "react-native";
+import type { Account } from "../../../../wallets/interfaces/wallet.js";
+import type { Theme } from "../../../core/design-system/index.js";
+import { spacing } from "../../design-system/index.js";
+import { CHECK, COPY_ICON } from "../icons/svgs.js";
+import { RNImage } from "./RNImage.js";
+import { ThemedText } from "./text.js";
+
+export type AddressProps = {
+  account: Account;
+  theme: Theme;
+  addressOrENS: string;
+};
+
+export const Address = ({ account, theme, addressOrENS }: AddressProps) => {
+  const [copySuccess, setCopySuccess] = useState(false);
+  useEffect(() => {
+    if (copySuccess) {
+      setTimeout(() => {
+        setCopySuccess(false);
+      }, 2500);
+    }
+  }, [copySuccess]);
+  return (
+    <TouchableOpacity
+      style={styles.addressContainer}
+      onPress={() => {
+        // @ts-ignore - missing native types
```

Comment:
prefer `@ts-expect-error`

```suggestion
        // @ts-expect-error - missing native types
```